src/github.ts

```diff
@@ -164,4 +164,19 @@ export const addPrComment = async (prNumber: number, comment: string) => {
       json.map((c: { body: string }) => c.body)
     }`,
   );
+
+// trigger GitHub action using workflow_dispatch
+export const triggerBackportAction = async () => {
+  const response = await fetch(
+    `${GITHUB_API}/repos/GiteaBot/gitea-backporter/actions/workflows/backport.yml/dispatches`,
```

Comment:
Did this line exist before?

Reply:
It did, but was pointing to your repo.

https://github.com/yardenshoham/gitea-backporter/pull/56/files#diff-a248ff5ec76368168c745273fee0601b243545fdca37aa3bec76ef9e9365fdb7L155

Reply:
So how did it trigger this repo?